backend/feed_service.py

```
@@ -27,12 +30,623 @@
+        "OPML import: Invalidated caches for tabs: %s.",
+        affected_tab_ids_set,
+    )
+
+
+def import_opml(opml_file_stream, requested_tab_id_str):
+    """Imports feeds from an OPML file, sending progress via SSE."""
+    root, error_resp = _parse_opml_root(opml_file_stream)
+    if error_resp:
+        return None, error_resp
+
+    total_feeds_to_import = _count_feeds_in_opml(root)
```

Ruff (0.14.14)

Local variable `total_feeds_to_import` is assigned to but never used

Remove assignment to unused variable `total_feeds_to_import`

(F841)